packages/amplify-go-function-runtime-provider/src/runtime.ts

```diff
@@ -95,12 +95,14 @@ export const buildResource = async ({ buildType, srcRoot, lastBuildTimeStamp }:
 
     if (isWindows) {
       envVars.CGO_ENABLED = 0;
+      executeCommand(['go.exe', 'install', 'github.com/aws/aws-lambda-go/cmd/build-lambda-zip@latest'], true, envVars, srcDir);
```

Comment:
can you explain the changes to this file?

Reply:
It's in aws-lambda-go doc https://github.com/aws/aws-lambda-go

Reply:
this is actually passing go.exe as an argument to go command and therefore breaking it

Reply:
submitted: https://github.com/aws-amplify/amplify-cli/pull/13789